src/placeos-frontend-loader/api/remote/remote.cr

```diff
@@ -23,7 +23,8 @@ module PlaceOS::FrontendLoader
       getter tag : String | Nil
       getter repo_path : String
 
-      def initialize(@repo_name : String, @branch : String? = "master", @tag : String? = nil, @hash : String? = "HEAD", @repo_path : String = "/")
+      def initialize(@url : String, @branch : String? = "master", @tag : String? = nil, @hash : String? = "HEAD", @repo_path : String = "/")
```

Comment:
I think `@repo_path` should be removed here

Reply:
You could use the `repo_name` instead,and save that as`repo_name`and `organisation`